cypress-io/cypress
# feat: remove protocol environment variable from binary

### Additional details


Remove the local protocol environment variable from the binary at binary build time. This will prevent third party code from being utilized in the place of our protocol.

### Steps to test


Run:

```
CYPRESS_LOCAL_PROTOCOL_PATH=~/protocol/cypress-services/packages/app-capture-protocol/dist/index.js CYPRESS_CONFIG_ENV=staging DEBUG=cypress:*protocol* yarn cypress run --record
```

And ensure that the protocol uses the downloaded script instead of the local script

### How has the user experience changed?


n/a

### PR Tasks


- [na] Have tests been added/updated?
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
